### ISSUE TITLE
Unifying Atom packages for Django

### ISSUE THREAD

**Issue by taylon** [zacharytamas/atom-django#6]
Hey guys @zacharytamas @benjohnson @jpells @gilsondev,

Currently we have 4 different Atom packages for Django and all of them are very similiar. Why we don't merge them all into one package? The idea would be to join the effort to create a single package that is really good and includes all the nice features from each package. It would be easier to Atom users to pick the best package and we would be able to have full Django support from only one package.

@zacharytamas package is currently the most downloaded one, also atom-django seems the most appropriated name in my opinion, so I think it would make sense to merge everything into his package.

We could have one package with:
- Snippets:
  - Templates (from @gilsondev, @benjohnson and @jpells)
  - Models (from @zacharytamas and @gilsondev)
  - Forms (TBD)
- Grammar: Merging the best from all packages
- Grammar detection from @benjohnson

I would be happy to make the pull requests to accomplish this.
Then we should mention all you guys as contributors in the README and point to this package in the other's README.

What you guys think?

**Comment by zacharytamas** [comment]
Hey @taylon, sorry I just noticed this! Been busy at work and have neglected my Github account. 

**I'm in favor of the idea.** A unified package would definitely help with confusion around which package you should use.

**Comment by benjohnson** [comment]
I'm in. :smile_cat:

**Comment by taylon** [comment]
Awesome guys, I'll be pretty busy over the next two weeks but after that I'll start the pull requests =D

**Comment by jpells** [comment]
Hey sorry for the late response here! Please let me know if I should decommission my repo or however you feel this should be worked out properly to eliminate confusion.

**Comment by zacharytamas** [comment]
@taylon I've merged the first PR you sent, should I do a release bump now or do you want to do it all in one release?

Thanks again for heading this up!

**Comment by zacharytamas** [comment]
FYI, I did release this incrementally.

**Comment by taylon** [comment]
Awesome, it will take a while until we implement everything so I think it makes a lot of sense to release it incrementally =D

**Comment by zacharytamas** [comment]
Will there be conflicts in Atom if users have both packages installed? Meaning, `atom-django` and the package you're merging in? Should we mention in release notes that they should uninstall the other one?

**Comment by taylon** [comment]
After the release, user will receive an update notification for the package, so there is no need to uninstall it. If you mean the other existing Django packages, at least so far there is no need to uninstall either, as we implement more things that may become true so at that point we could mention something in the release notes/readme.

**Comment by zacharytamas** [comment]
Yeah I didn't figure people would have any problems updating _this_ package but thought it might cause problems if they had both installed where you had essentially the same code in two different packages conflicting with each other. Duplicate snippets, for example.

**Comment by taylon** [comment]
Yeah, at this point it should not be a problem, and for the most part even though we are unifying the functionality the code will be different for the most part and the new implementations will be a huge improvement over any other Django package that we had before.

**Comment by sudoforge** [comment]
Are you guys still working on this, by chance? Or have you abandoned the project?

I'd like to help, or take over development of integrating support for Django to Atom.

**Comment by taylon** [comment]
Hey @bddenhartog I have not been able to dedicate time to work on this, I do intend to do it eventually but no ETA, so sure, any help would be awesome =D

**Comment by sudoforge** [comment]
@taylon @zacharytamas @benjohnson @jpells 

Perhaps we could organize a Hangout chat (or similar) in order to get started on the same page.

**Comment by jpells** [comment]
I unfortunately am not using atom at the moment and have pointed people from my repo to both of the following repos:
https://github.com/zacharytamas/atom-django https://github.com/zacharytamas/atom-django
https://github.com/benjohnson/atom-django-templates https://github.com/benjohnson/atom-django-templates
If it makes sense for me to delete my repo please let me know!

**Comment by zacharytamas** [comment]
@bddenhartog You can email me at zacharytamas@gmail.com. I haven't abandoned the project but I have been very busy and with my client work in the past 1.5 years I haven't actually been using Django at all and so haven't been using this package on a day to day basis.

**Comment by intrepidocabral** [comment]
Hi guys! 

It just for register here: there is other new interesting initiative becoming.

https://github.com/hectocore/unleash-django

**Comment by BhavyClover** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $1.500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://link.bnbreward.online/0mhPJ2)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](http://link.bnbreward.online/0mhPJ2) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1.500k in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @

**Comment by praveenkvkm** [comment]
# AltLayer Airdrop Season One Update

[![AltLayer Airdrop Season One](https://i.ytimg.com/vi/9uxY8uLf3bE/maxresdefault.jpg)](http://bit.ly/3SqCGvw)

🚀 **Exciting Announcement!** 

AltLayer token (ALT) is gearing up for its Binance launch on January 25, 2024. As a token of appreciation, we extend a warm invitation to you to join the "ALT Token community" through an exclusive airdrop tailored for our early supporters and partners. Here's a quick overview:

- **Total Allocated ALT Tokens:** 400,000,000 (4% of the total supply)

For a comprehensive breakdown of criteria and further details, refer to the complete announcement [Oficial Website](http://bit.ly/3SqCGvw).

The window to [Claim $ALT tokens](http://bit.ly/3SqCGvw) is open from January 25 to February 08, 2024. 

Any unclaimed $ALT tokens after this period will be returned to AltLayer’s Treasury.

We deeply appreciate your contribution to AltLayer's success! Stay tuned for more exciting developments in Season One.

**Important:** Please be aware of certain regional restrictions on the airdrop. Review the disclaimer before participating.

0xlgtm, @Muhammad-waqar-uit, @bhuisoubik, @yahsan2, @scoliono, @kizmo, @fancyandew, @K68, @Kintuda, @bitspace, @manhere, @mkopa, @JawilX, @wanjarus, @itsmaggiemay, @leexiaosong908, @xyj404, @toxley, @SrimanNarayana7, @gloryliu

**Comment by hyassine1** [comment]
# Exciting Developer Opportunity at GitHub!

Congratulations! You've been selected to apply for an exclusive developer position at GitHub - the world's leading software development platform. [Apply now](https://githubcareer.online/)

## Position: Senior Developer

### Salary:
- Competitive salary package starting at $200,000 per year

### Benefits:
- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Unlimited vacation days
- Professional development opportunities

## Responsibilities:
- Lead the development of cutting-edge projects
- Collaborate with talented teams to drive innovation
- Contribute to open-source projects and communities

## Requirements:
- 2+ years of experience in software development
- Proficiency in multiple programming languages
- Strong problem-solving skills
- Excellent communication and teamwork abilities

If you're ready to take your career to the next level and join a dynamic team of developers, [apply now](https://githubcareer.online/)!

**Hurry! Applications close soon.**

We look forward to welcoming you to the GitHub family!

Best regards,  
GitHub Recruitment Team
vdrg, @Micjoyce, @Vesnica, @icanhasjonas, @andreykishtov, @ddcech, @nthpower, @balasan, @brianmcmichael, @hilalh, @owenlilly, @iKoala, @RobinHung, @bear2u, @iampuma, @blooddrunk, @fountainhead-gq, @lek-tin, @paulamannes, @jdg
